epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -94,6 +94,11 @@ String getterCode() {
     return isPrivate ? getterMethodName + "()" : name;
   }
 
+  // Special case to avoid generating recursive getter if field and its getter names are the same
+  String superGetterCode() {
```

Comment:
good point. I guess this is fine for now, if we have problems in the future we can refactor then